examples/browser-tools/browser-tool-helpers.ts

```diff
@@ -0,0 +1,906 @@
+import type { SearchRequest, SearchResponse, CrawlRequest, CrawlResponse } from 'ollama'
+
+// Local types used only by this example
```

Comment:
```suggestion
// Local types used only by this example
```

can you do another pass to get rid of most comments - only add if necessary